src/xcomponentAPI.ts

```diff
@@ -23,12 +23,12 @@ class XComponentAPI {
         }).bind(this));
     };
 
-    createSession(xcApiFileName: string, serverUrl: string, createSessionListener: (error: Error, session: Session) => void, reconnectionAfterError: boolean = false, reconnectionIntervalSeconds: number = 10): void {
-        this.connection.createSession(xcApiFileName, serverUrl, createSessionListener, reconnectionAfterError, reconnectionIntervalSeconds);
+    createSession(xcApiFileName: string, serverUrl: string, createSessionListener: (error: Error, session: Session) => void, deconnectionErrorListener?: (closeEvent: CloseEvent) => void): void {
```

Comment:
This error parameter is a variable not a callback. It used when you cant establish connection at the first place. deconnectionErrorListener is a function called when the connection is interrupted suddenly after is was esthablished in the first place.